src/fft_small/profile/p-fft_small_vs_gmp.c

```diff
@@ -0,0 +1,60 @@
+/*
+    Copyright (C) 2023 Fredrik Johansson
+    Copyright (C) 2024 Albin Ahlbäck
+
+    This file is part of FLINT.
+
+    FLINT is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.  See <https://www.gnu.org/licenses/>.
+*/
+
+#include "mpn_extras.h"
+#include "fft_small.h"
+#include "profiler.h"
+
+#define N_MAX 4000
+
+int main(void)
+{
+    mp_ptr x, y, r, s;
+    slong n;
+
+    x = flint_malloc(sizeof(mp_limb_t) * N_MAX);
+    y = flint_malloc(sizeof(mp_limb_t) * N_MAX);
+    r = flint_malloc(2 * sizeof(mp_limb_t) * N_MAX);
+    s = flint_malloc(2 * sizeof(mp_limb_t) * N_MAX);
+
+    mpn_random2(x, N_MAX);
+    mpn_random2(y, N_MAX);
+
+    flint_printf("mpn_mul_n vs flint_mpn_mul_n\n\n");
+
+    /* for (n = 900; n <= N_MAX; n = 1.02 * n) */
+    for (n = 1500; n <= 1600; n += 5)
+    {
+        double t1, t2, FLINT_SET_BUT_UNUSED(__);
+
+        flint_printf("n = %4wd: ", n);
+
+        TIMEIT_START
+        mpn_mul_n(r, x, y, n);
+        TIMEIT_STOP_VALUES(__, t1)
+
+        TIMEIT_START
+        mpn_mul_default_mpn_ctx(s, x, n, y, n);
```

Comment:
Please make sure that ``make profile`` works on machines that don't have this function.

Reply:
Thanks for observing this.